### DISCUSSION TITLE
Share data between servers

### DISCUSSION THREAD

**Discussion by DiogoMurano** [jeremylvln/Shulker#645] [Ideas]
Would it make sense to have some feature to send data between servers?

For example, match data, whether a match has started or not, and things like that

**Comment by jeremylvln** [comment]
Shulker is meant for infrastructure management. If one needs to pass data across servers, they could do it with their own plugins. 

I really want to avoid cluttering Shulker with too specific use-cases.

Hope it makes sense!

**Reply by DiogoMurano** [reply]
Make sense! Perfect
